### PR TITLE
added a timer just above the play/pause button and progress bar to tr…

### LINKED ISSUE #46: Display current playtime
A simple text piece in the audio player which can display `music.currentTime : music.duration` should suffice.

### PR DESCRIPTION
**Pull request by Jason9199**
…ack the current play time and the total duration time

Fixes #46 

#### This PR is for [Anime Name].

- [ x ] This issue is for enhancement/bug fixing.
- [ x ] I have checked my addition locally and it is working and won't break existing entries.
- [ x ] I have gone through [Contribution Guidelines](https://github.com/Anshuman-Verma/ongaku/blob/master/CONTRIBUTING.md).
- [ x ] This pull request will be closed if I failed to update it even once in a week.

### CODE REVIEW

File: js/ongaku.js [hunk 1]
```diff
@@ -41,6 +42,11 @@ window.addEventListener("load", function () {
 			play();
 		}
 		playHead.style.marginLeft = (percent / 100) * (timeLine.offsetWidth - playHead.offsetWidth)  + "px";
+		var currentTimeMin = ("0" + parseInt(music.currentTime / 60)).slice(-2);
```
Comment (anshumanv) [inline js/ongaku.js]: Could you please use let instead of var?

**Comment by anshumanv** [comment]
The changes look great !
Apparently the current HTML player is very basic and we are looking to port it to a better player with good UI/UX and features in #175 , perhaps do you think you can help with that ?

**Comment by anshumanv** [comment]
Also one thing that i observed is: the playtime is getting half-cut when the player is down which doesn't look right, can you do something to make it completely visible when the player is down ?

**Comment by Jason9199** [comment]
@anshumanv I would be happy to make the changes you requested. I apologize for the delay, got extra busy this past week. I will get this accomplished ASAP.

**Comment by anshumanv** [comment]
No problem, go for it !

**Review by anshumanv — APPROVED** [review]
👍

**Pull request merged**